data_steward/cdr_cleaner/cleaning_rules/fix_unmapped_survey_answers.py

```diff
@@ -0,0 +1,203 @@
+"""Original Issues: DC-1043, DC-1053 
+
+PPI answers are mapped to standard answer concepts in concept_relationship through 'Maps to value' (it has been this 
```

Comment:
I realize pylint passes, but should [max line length be 100](https://github.com/all-of-us/curation/blob/develop/.pylintrc#L371)? I configured my editor so it complains about this. Let me know if I'm wrong- I'm happy to start using lines that are a little longer. :)

Reply:
Will do